park_api/cities/Bonn.py

```diff
@@ -21,12 +21,15 @@ def parse_html(html):
     
     for row in soup.find_all("div", class_='parking-lots'):
       for column in row.find_all("div", class_='wpb_column vc_column_container vc_col-sm-3'):
-        print(column)
         h3 = column.find_all("h3")
-        print(h3)
         if not h3[0].a == None:
           name = h3[0].a.string
           lot = geodata.lot(name)
+          ltype = None
+          if "Parkhaus" in name:
+            ltype = "Parkhaus"
+          elif "Parkplatz" in name:
+            ltype="Parkplatz"
```

Comment:
None (which shouldn't break anything since lot_type is not required), so a lot type is only set if known.